src/app/dashboard/dashboard.component.ts

```diff
@@ -13,16 +13,21 @@ export class DashboardComponent implements OnInit {
     {name: 'Two'},
     {name: 'Three'}
   ];
-  // createNewDashboardDialog: MdDialog;
-  constructor(public createNewDashboardDialog: MdDialog) {
-    this.ngOnInit();
+  _createNewDashboardDialog: MdDialog;
```

Comment:
* The dialog should be renamed `_newDashboardDialog`
* Currently, this add button is Add **tab** button. The new dashboard dialog should be opened by clicking add button in dashboard management component.